app/models/ext_management_system.rb

```diff
@@ -143,6 +143,7 @@ def hostname_uniqueness_valid?
   virtual_column :total_vms_never,         :type => :integer
   virtual_column :total_vms_suspended,     :type => :integer
   virtual_total  :total_subnets,           :cloud_subnets
+  virtual_column :v_supports_block_storage, :type => :boolean
```

Comment:
IIRC, I think we were trying to prevent adding further `v_` methods, so this could be `:supports_block_storage`